python/social_bot/tasks.py

```diff
@@ -338,7 +348,14 @@ def task_specific_observation(self, agent):
             observation besides self states, for the non-image case
         """
         goal = self._world.get_model(self._goal_name)
-        return np.array(goal.get_pose()[0]).flatten()
+        pose = np.array(goal.get_pose()[0]).flatten()
+
+        for name in self._additional_observation_list:
+            obj = self._world.get_model(name)
+            obj_pose = np.array(obj.get_pose()[0]).flatten() 
```

Comment:
we can the name "obj_pos = np.array(obj.get_pose()[0]).flatten()" here because pose includes rotation, and pos is only the position part.

Reply:
It is now renamed to "obj_pos = np.array(obj.get_pose()[0]).flatten()" in the new commits.